routes/delius.js

```diff
@@ -24,17 +87,16 @@ router.get('/staff/staffCode/:staffCode/managedOffenders', (req, res) => {
 })
 
 router.get('/offenders/nomsNumber/:nomsNumber/allOffenderManagers', (req, res) => {
-  const { nomsNumber } = req.params
   const ros = [
     {
       isPrisonOffenderManager: false,
       isUnallocated: false,
       isResponsibleOfficer: true,
-      staff: { forenames: 'Ryan', surname: 'Orton'},
+      staff: { forenames: 'Ryan', surname: 'Orton' },
       staffCode: 'DELIUS_ID',
-      team: { localDeliveryUnit: { code: 'lduCode', description: 'lduDescription' } },
-      probationArea: { code: 'probationAreaCode', description: 'probationAreaDescription' },
```

Comment:
I originally made sure these matched the original data from the responsibleOfficers endpoint - i dont think the feature tests will actually use it though so it shouldn't matter, and linking it up makes more sense anyway